### PR TITLE
Cleanup analysis warnings

### PR DESCRIPTION
**Pull request by jpnurmi**

### CODE REVIEW

File: packages/subiquity_client/lib/src/types.dart [hunk 1]
```diff
@@ -4,6 +4,8 @@ import 'package:json_annotation/json_annotation.dart';
 part 'types.freezed.dart';
 part 'types.g.dart';
 
+// ignore_for_file: invalid_annotation_target
```
Comment (jpnurmi) [inline packages/subiquity_client/lib/src/types.dart:7]:
```
info • The annotation 'JsonKey' can only be used on fields or getters • lib/src/types.dart:58:6 • invalid_annotation_target
...
```

https://pub.dev/documentation/freezed/latest/#disabling-invalid_annotation_target-warning-and-warning-in-generates-files

File: packages/subiquity_client/lib/src/http_unix_client.dart [hunk 2]
```diff
@@ -103,7 +103,6 @@ class HttpUnixClient extends BaseClient {
     return req.completer.future;
   }
 
-  @override
```
Comment (jpnurmi) [inline packages/subiquity_client/lib/src/http_unix_client.dart:106]:
```
info • The method doesn't override an inherited method • lib/src/http_unix_client.dart:107:16 • override_on_non_overriding_member
```

File: packages/subiquity_client/lib/src/http_unix_client.dart [hunk 3]
```diff
@@ -16,14 +16,14 @@ enum _HttpParserState {
...
+  String httpVersion = '';
+  int statusCode = 0;
+  String reasonPhrase = '';
+  Map<String, String> headers = <String, String>{};
 
   BaseRequest request;
-  var completer = Completer<StreamedResponse>();
-  var stream = StreamController<List<int>>();
+  Completer<StreamedResponse> completer = Completer<StreamedResponse>();
+  StreamController<List<int>> stream = StreamController<List<int>>();
```
Comment (jpnurmi) [inline packages/subiquity_client/lib/src/http_unix_client.dart:26]:
```
info • Type annotate public APIs • lib/src/http_unix_client.dart:19:7 • type_annotate_public_apis
...
```

File: packages/ubuntu_desktop_installer/lib/pages/installation_complete/installation_complete_page.dart [hunk 4]
```diff
@@ -1,5 +1,3 @@
-import 'dart:io' as io;
```
Comment (jpnurmi) [inline packages/ubuntu_desktop_installer/lib/pages/installation_complete/installation_complete_page.dart:1]:
```
info • Unused import: 'dart:io' • lib/pages/installation_complete/installation_complete_page.dart:1:8 • unused_import
```

File: packages/ubuntu_wizard/test/system_shutdown_test.dart [hunk 5]
```diff
@@ -2,8 +2,8 @@ import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:mockito/mockito.dart';
 import 'package:subiquity_client/subiquity_client.dart';
-import 'package:ubuntu_wizard/src/utils/system_shutdown.dart';
 import 'package:ubuntu_test/mocks.dart';
+import 'package:ubuntu_wizard/src/utils/system_shutdown.dart';
```
Comment (jpnurmi) [inline packages/ubuntu_wizard/test/system_shutdown_test.dart:6]:
```
info • Sort directive sections alphabetically • test/system_shutdown_test.dart:6:1 • directives_ordering
```

**Comment by oSoMoN** [comment]
LGTM, thanks!

**Pull request merged**